### PR TITLE
Case study (EMF)

### PR DESCRIPTION
**Pull request by Cleop**
#291 - EMF Case Study page. Including: title, text and phone images

### CODE REVIEW

**Comment by Cleop** [comment]
@SimonLab @iteles @nelsonic - are you able to review or merge this PR? Thanks :blush:

**Comment by nelsonic** [comment]
@Cleop I'm happy with the content (_already approved above_). 👍 
if @iteles & @SimonLab are _unavailable_ to review/merge by end of today, I'm happy to merge. ⌛️

File: redesign/cs-emf.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        <img src="./img/common/emf-phone-2.png" alt="A phone displaying an app on a page for selecting tags" class="dn db-ns">
+        <img src="./img/common/emf-phone-3.png" alt="A phone displaying an on a page to create a new challenge" class="dn db-ns">
+      </div>
+
+      <article class="w-50-ns dtc v-top istok-web lh-copy dwyl-dark-gray">
+        <h3 class="b mb2">Project: <span class="normal">CE100</span></h3>
+        <div class="mb4">
+          <a href="#" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
+            <img src="./img/common/yellow-gray-heart.png" alt="dwyl heart logo" class="h1">
+            Live App
```
Comment (SimonLab) [inline redesign/cs-emf.html]: I'm not sure if we want to provide a link to the live application, @iteles any thoughts?

File: redesign/cs-emf.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+          </a>
+          <a href="#" class="dib link mv2 mh1 pa2 dwyl-bg-dark-gray white br2 f5 shadow-4">
```
Comment (SimonLab) [inline redesign/cs-emf.html]: link to the repo: https://github.com/emfoundation/ce100-app

File: redesign/cs-emf.html (hunk 1 above)
Comment (markwilliamfirth) [inline redesign/cs-emf.html]: I think we should do this, prospective clients want to see our previous work - any reasons why you think we shouldn't @SimonLab ?

**Comment by markwilliamfirth** [comment]
Merging as approved

@SimonLab can you open your comments [here](https://github.com/dwyl/dwyl-site/pull/292#pullrequestreview-49770376) as new issues please?

**Pull request merged**